### ISSUE TITLE
Incorrect package naming

### ISSUE THREAD

**Issue by schneider8357** [ashleysommer/sanic-plugin-toolkit#22]
When I run pip freeze with `sanic-plugin-toolkit`  installed, it actually prints out `sanic plugin toolkit==1.0.1` (notice the missing dashes).
Tested with python 3.8.10 and pip versions 21.2.3 and 21.2.4.

The output includes:
 ```
sanic==21.6.2
sanic plugin toolkit==1.0.1
Sanic-Cors==1.0.0
sanic-jwt==1.7.0
sanic-openapi==21.6.1
sanic-routing==0.7.1
```

**Comment by ashleysommer** [comment]
Thanks for letting me know. I'll look into it.

**Comment by ashleysommer** [comment]
@schneider8357 This is fixed in SPTK v1.1.0
fixed by https://github.com/ashleysommer/sanic-plugin-toolkit/commit/87b7f8478748591be57751b7e8e8e93a0c83ac22
